sdk/contentsafety/azure-ai-contentsafety/src/test/java/com/azure/ai/contentsafety/AnalyzeImageTests.java

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.ai.contentsafety;
+
+import com.azure.ai.contentsafety.models.*;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public final class AnalyzeImageTests extends ContentSafetyClientTestBase {
+    @Test
+    public void testAnalyzeImageTests() throws IOException {
+        // method invocation
+        ImageData image = new ImageData();
+        String cwd = System.getProperty("user.dir");
+        String source = "/src/test/resources/image.jpg";
```

Comment:
If it is just same file, please keep only one. You can load it from samples folder.